### PR TITLE
MMCA-4155 : Update performance tests for repos

### PR DESCRIPTION
**Pull request by sameerhmrc**
JIRA Link : https://jira.tools.tax.service.gov.uk/browse/MMCA-4155

### CODE REVIEW

**Comment by sameerhmrc** [comment]
customs guarantee account pt

File: .gitignore [hunk 1]
```diff
@@ -24,4 +24,5 @@ target
 lib_managed
 src_managed/
 project/project
-project/target
\ No newline at end of file
+project/target
+/.bsp
```
Comment (AdamCoops) [inline .gitignore:28]: There is a no new line at end of file error here.

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.6.2
\ No newline at end of file
+sbt.version=1.7.2
```
Comment (AdamCoops) [inline project/build.properties:1]: There is a no new line at end of file error here.

File: project/plugins.sbt [hunk 3]
```diff
@@ -3,8 +3,8 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
...
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.8.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.9.0")
 
-addSbtPlugin("io.gatling" % "gatling-sbt" % "3.2.1")
+addSbtPlugin("io.gatling" % "gatling-sbt" % "4.1.5")
 
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
\ No newline at end of file
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.0")
```
Comment (AdamCoops) [inline project/plugins.sbt:10]: There is a no new line at end of file error here.

**Review by AdamCoops — APPROVED** [review]

**Pull request merged**